docs/fundamentals/apicompat/preview-apis.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Preview APIs
+description: Provides an understand how APIs can be marked as preview.
+ms.date: 4/4/2024
+---
+
+# Preview APIs
+
+The .NET platform prides itself with taking compatibility very seriously. As
+such, the the library ecosystem tends to avoid making breaking changes,
+especially with respect to the API.
+
+Nonetheless, when designing APIs it's important to collect feedback from users
+and change the API based on their feedback. To avoid surprises, it's important
+that users understand which APIs are considered stable and which ones are still
+in active development and might change.
```

Comment:
```suggestion
Nonetheless, when designing APIs, it's important to collect feedback from users and change the API based on their feedback. To avoid surprises, it's important that users understand which APIs are considered stable and which ones are still in active development and might change.
```